docs/nonlocal-networks.rst

```diff
@@ -0,0 +1,138 @@
+
+To connect to any other network that has been defined in ``brownie-config.json``, you must specify the `--network` flag when launching Brownie as below:
+::
+    $ brownie --network ropsten
+
+
+Each time Brownie is loaded, it will first attempt to connect to the ``host`` address to determine if the RPC client is already active.
+
+Using the ``brownie.netowrk`` methods
+-------------------------------------
+
+You can interact with any network defined in ``brownie-config.json`` programatically using the ``brownie.network.main`` methods.
+
+Connecting to a network:
+
+.. code-block:: python
+
+    >>> network.connect('ropsten')    
+    >>> network.is_connected()
+    True
+    >>> network.show_active()
+    'ropsten'
+
+Disconnecting from a network:
+
+.. code-block:: python
+
+    >>> network.disconnect()
+    >>> network.is_connected()
+    False
+
+Interacting with non-local networks
+==============================
+
+``brownie.module.rpc``
+--------
+
+The :ref:`rpc` module is unavailable when working with non-local networks.
+
+Accounts
+--------
+
+When loading an account for interacting with a non-local network, you must provide the private key when loading the account in order to be able to sign transactions or deploy contracts
+
+.. code-block:: python
+
+        >>> accounts.add('8fa2fdfb89003176a16b707fc860d0881da0d1d8248af210df12d37860996fb2')
+        <Account object '0xc1826925377b4103cC92DeeCDF6F96A03142F37a'>
+        >>> accounts[0].balance()
+        17722750299000000000
+
+Once an account is added to the account object, the ``accounts.save`` and ``accounts.load`` can be used to save the accounts to an encrypted keystore and then load for later use.
+
+Transactions
+------------
+
+* Transaction status
+When submitting a transaction on non-local networks, blocks are not immediately so transactions will likewise not be immediately confirmed. 
+A :ref:`api-network-tx` object is provided immediately and can be stored to unique variables though ``TransactionReceipt.status`` will be ``-1`` until the transaction is mined and either succeeds or reverts.  
+
+* Debugging 
+The Brownie :ref:`debug` tools rely upon `debug_traceTransaction <https://github.com/ethereum/go-ethereum/wiki/Management-APIs#user-content-debug_tracetransaction>`__ RPC method which is not supported by `Infura <https://infura.io>`__. Attempts to call it will result in a ``RPCRequestError``.
+This means that the below ``TransactionReceipt`` attributes and methods are unavailable:
+
+* ``TransactionReceipt.return_value``
+* ``TransactionReceipt.trace``
+* ``TransactionReceipt.call_trace``
+* ``TransactionReceipt.traceback``
+* ``TransactionReceipt.source``
+
+:ref:`api-network-contract`
+---------
+On non-local networks, use the ``Contract`` class to interact with deployed contracts.  ``ContractContainer`` and ``ProjectContract`` are unavailable as these are only used with the local   
```

Comment:
A project can be deployed to a non-local network, and then ``ProjectContract`` instances will be available for that network.

* Calling `ContractContainer.at` is useful for obtaining `ProjectContract` instances, to interact with deployed contracts that are part of the project
* Instantiating a `Contract` object is useful for interacting with contracts that are not of a project

Reply:
I've updated this section in line with this. Let me know if it's accurate.